src/app/docs/definition-list/definition-list-docs.component.html

```diff
@@ -0,0 +1,19 @@
+<sky-docs-demo-page
+  moduleName="SkyDefinitionListModule"
+  moduleSourceCodePath="src/app/public/modules/definition-list/"
+  pageTitle="Definition List"
+>
+  <sky-docs-demo-page-summary>
+    The definition list module displays a list of label-value pairs.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section
+      heading="UX guidelines"
+    >
+      <p>
+        Don't append colons to definition list labels.
+      </p>
+    </sky-docs-demo-page-section>
+  </sky-docs-design-guidelines>
```

Comment:
Seems like we could add a Related components section for alert and toolbar since we reference those components elsewhere in the docs.